requirements.txt

```
@@ -3,4 +3,5 @@ tensorflow_probability
 numpy
 torch
 torchvision
-gym
\ No newline at end of file
+gym
```

```suggestion
gymnasium
```
Package name changed